charts/radar-output/values.yaml

```diff
@@ -37,10 +37,10 @@ resources:
   # resources, such as Minikube. If you do want to specify resources, uncomment the following
   # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
   limits:
-    cpu: 500m
+    cpu: 1000m
   requests:
     cpu: 100m
-    memory: 128Mi
+    memory: 400Mi
```

Comment:
Note that `Xmx` is `3g`. The request should also be for that size.

Reply:
Isn't the Xmx for maximum amount of memory?

Reply:
Indeed, it's just that if you would request for a lower amount, there is a very large possibility that in fact the  output generation will consistently need more than the requested amount.

Reply:
I've tried to intentionally keep the memory request as low as possible, low enough that the application can start because if the request is too high the pod will not be scheduled by Kubernetes. And these are some base values to start with and not indication on actually how much memory they'll use. For example right now on production instance of Radar AD it's using 472Mi while on development server of CHDR it's using 4166Mi.